tests/framework/PostProcessors/InterfacedPostProcessor/RiskMeasuresTimeDep/testPrintHistorySet_dump_0.csv

```diff
@@ -0,0 +1 @@
+Astatus,Bstatus,time
1,1,0
1,1,0.5
1,1,1
1,1,1.5
1,1,2
1,1,2.5
1,1,3
1,1,3.5
1,0,4
1,0,4.5
1,0,5
1,0,5.5
1,0,6
1,0,6.5
1,0,7
1,0,7.5
1,0,8
0,1,8.5
0,1,9
0,1,9.5
0,1,10
0,1,10.5
0,1,11
0,1,11.5
0,1,12
0,0,12.5
0,0,13
0,0,13.5
0,0,14
0,0,14.5
0,0,15
0,0,15.5
0,0,16
```

Comment:
Are these Windows newlines? Can we change this file to have UNIX newlines and not have it show up like this in github?